main.go

```diff
@@ -86,6 +87,14 @@ func mainCore() int {
 		time.Sleep(250 * time.Millisecond)
 	}()
 
+
+	//Initialize the dcrwallet funcs for accesing the dcr wallet daemon
+	//This creates its own RPC connection since the wallet daemon
+	//Listens on a different port from the dcrd RPC server
```

Comment:
Please reformat comment `//Initialize` --> `// Initialize`, and join the third line with the second line and wrap as appropriate